src/lib.rs

```diff
@@ -178,13 +178,14 @@ pub mod prelude {
         JavaScriptCodeWithScope as BsonJavaScriptCodeWithScope, Regex as BsonRegex,
         Serializer as BsonSerializer, Timestamp as BsonTimestamp,
     };
+    //See driver pull request change for errors: https://github.com/mongodb/mongo-rust-driver/pull/1102
```

Comment:
question: What is the purpose of this comment?

Reply:
This is where i was able to trace back why WriteBulkFailure enum didn't exist and has been replaced in the driver from the pull request. There are a lot of changes from the the driver. i can remove it and put it in the PR description

Reply:
Yes, this comment is only useful where comparing the diff, but we don’t need it when looking at the new code

Reply:
I remove it and reference it from the PR description